app/javascript/components/filters/Filters.vue

```diff
@@ -49,18 +54,26 @@
 
         // create an empty array for each filter
         this.filters.forEach(filter => {
+
           if (filter.name !== undefined && filter.options.length > 0) {
+            const preselected = this.presetOptionsForFilter(filter.name)
+
             let obj = {}
 
             obj.name = filter.name
-            obj.options = []
+            obj.options = preselected
             obj.type = filter.type
 
             array.push(obj)
           }
         })
 
         this.$store.dispatch('table/setFilterOptions', array)
+      },
+
+      presetOptionsForFilter(filterName) {
+        const preselected = this.presetFilters.find(presetFilter => presetFilter.name === filterName)
+        return preselected ? preselected?.options : []
```

Comment:
Re: https://github.com/unepwcmc/action-agenda/pull/79#discussion_r840596724, this should actually be `return preselected?.options || []` - the `?.` part resolves to undefined if `preselected` is falsey, or `preselected.options` if not. If it's undefined, because that's falsey it returns the other side of the `||`